packages/react-aria-components/stories/animations.stories.tsx

```diff
@@ -0,0 +1,41 @@
+
+import {
+  Button,
+  Dialog,
+  DialogTrigger,
+  Heading,
+  Modal,
+  ModalOverlay
+} from 'react-aria-components';
+import React from 'react';
+import styles from './styles.css';
+
+
+export default {
+  title: 'React Aria Components - Animations'
```

Comment:
This seems like a weird place for. Are we planning for more RAC animations? Shouldn't this be under RAC?

Reply:
Yeah, wasn't too sure where to put it, seemed out of place with the RAC stories, seemed out of place with our other stories. I think we can decide where to move it later. Maybe other animation related bugs will come in or maybe we should have a story group called "Bugs"